TODO/package.json

```diff
@@ -13,7 +13,12 @@
     "@react-native-async-storage/async-storage": "^1.17.10",
     "react": "18.0.0",
     "react-native": "0.69.5",
-    "react-native-vector-icons": "^9.2.0"
+    "react-native-uuid": "^2.0.1",
+    "react-native-vector-icons": "^9.2.0",
+    "react-redux": "^8.0.2",
+    "redux": "^4.2.0",
+    "redux-logger": "^3.0.6",
```

Comment:
In a production app, remove this logger otherwise this will impact the performance of the app